apps/hyperdrive-trading/src/ui/appconfig/useAppConfig.ts

```diff
@@ -5,10 +5,31 @@ import {
   sepoliaAppConfig,
 } from "@hyperdrive/appconfig";
 import assertNever from "assert-never";
+import { ZERO_ADDRESS } from "src/base/constants";
 import { SupportedChainId } from "src/chains/supportedChains";
-import { foundry, mainnet, sepolia } from "viem/chains";
+import { b3Sepolia } from "src/network/b3Sepolia";
+import { baseSepolia, foundry, mainnet, sepolia } from "viem/chains";
 import { useChainId } from "wagmi";
 
+const emptyAppConfig: AppConfig = {
+  chainId: 84532,
```

Comment:
This will make it easier to identify if somehow the empty gets passed directly through and expresses it's intent a bit better.
```suggestion
  chainId: 00000,
```